tests/src/test/scala/system/health/BasicHealthTest.scala

```diff
@@ -55,125 +55,24 @@ class BasicHealthTest
   val messageHubFeed = "messageHubFeed"
   val messageHubProduce = "messageHubProduce"
   val actionName = s"$messagingPackage/$messageHubFeed"
-
-  val consumerInitTime = 10000 // ms
-
-  val kafkaUtils = new KafkaUtils
-
   val maxRetries = System.getProperty("max.retries", "60").toInt
 
   behavior of "Message Hub feed"
 
-  it should "create a new trigger" in withAssetCleaner(wskprops) {
```

Comment:
Duplicate test, so deleting it. The next text in this suite tests the same functionality.